measures/ResStockArguments/measure.rb

```diff
@@ -783,6 +784,27 @@ def modify_setpoint_schedule(schedule, offset_magnitude, offset_schedule)
     end
     return schedule
   end
+
+  def convert_args(model, args)
+    measures_dir = File.absolute_path(File.join(File.dirname(__FILE__), '../../resources/hpxml-measures'))
+    full_measure_path = File.join(measures_dir, 'BuildResidentialHPXML', 'measure.rb')
+    measure = get_measure_instance(full_measure_path)
+
+    args.each do |name, value|
+      measure.arguments(model).each do |arg|
```

Comment:
Yes, this was the general approach I was suggesting. Nice. This should definitely reduce bugs going forward.

But why are you going through all those complication to get the measure arguments? You can just pass them in. See next comment...